code/modules/mob/living/carbon/xenomorph/castes/baneling/castedatum_baneling.dm

```diff
@@ -0,0 +1,166 @@
+/datum/xeno_caste/baneling
+	caste_name = "Baneling"
+	display_name = "Baneling"
+	upgrade_name = ""
+	caste_desc = "An alien bloated with chemicals and ready to explode."
```

Comment:
```suggestion
	caste_desc = "Gross, cute, bloated and ready to explode!"
```
remember, xeno_caste is basically examining xenos when you're point of view are the aliens. so it can have leeway in seriousness.